src/common-components/Logistration.jsx

```diff
@@ -16,7 +16,9 @@ import { LOGIN_PAGE, REGISTER_PAGE } from '../data/constants';
 import { updatePathWithQueryParams, getTpaHint } from '../data/utils';
 import { LoginPage } from '../login';
 import { RegistrationPage } from '../register';
+import BaseComponent from '../base-component';
 
+jest.mock('@edx/frontend-platform/auth');
```

Comment:
I guess this is added by mistake and needs to be removed?

Reply:
Yup. Copy/paste in wrong file. I will removed it. Thanks for pointing this.